dev/package.json

```diff
@@ -7,7 +7,7 @@
   "repository": "allenai/syrup",
   "license": "Apache-2.0",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
+    "test": "mocha --compilers js:babel-register **/*_test.js",
```

Comment:
ugly, but this is what you have to add to a project to enable mocha testing with all the babel goodies